frontend/device/pocketbook/device.lua

```diff
@@ -288,13 +288,6 @@ end
 
 -- Predicate, so no self
 function PocketBook.canAssociateFileExtensions()
-    local f = io.open(ext_path, "r")
-    if not f then return true end
```

Comment:
I don't believe these lines are mentioned in your description? But if it's intentional and desired, you can just say `canAssociateFileExtensions = yes` up above in `local PocketBook = Generic:new{` and remove this entire function.

Reply:
These extra lines basically do nothing, as the function would return `true` in any case if `#koreader` condition is dropped (file not found? ⇒ `true`; file empty? ⇒ `true`; file non-empty? ⇒ `true`)

Reply:
But I do agree about `canAssociateFileExtensions = yes`, that can be a better way to achieve the same result.